### ISSUE TITLE
Unable to take screenshot in afterEach hook

### ISSUE THREAD

**Issue by Matthematic** [cerner/terra-toolkit#733]
# Bug Report

## Description

I would like to take a screenshot in an afterEach hook so that I can reduce repeated code, however when I try this I get the error "Cannot read property 'file' of undefined". This is because of the following line: https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js#L106

## Steps to Reproduce

1. Run a spec with an afterEach() method of a describe() that takes a screenshot using the terra api. 

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/6292111/156001542-13bb1eb8-e47f-4b46-b1be-7d893a48356f.png)

## Expected Behavior

I expect the screenshot to succeed.

## Possible Solution

It appears these hooks hold the file path in `context.suite` instead, so I think the offending line of code above should be updated to check for that.

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):
